# -----------------------------------------------------------------------------
# Copyright 2015 Fraunhofer IPA
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# -----------------------------------------------------------------------------
# Configuration file for a real robot
# -----------------------------------------------------------------------------
loggerLevel: 1
host: 172.31.1.107
#host: localhost
port: 30002
isDummy: False
jointNames: ["shoulder_pan_joint", "shoulder_lift_joint", "elbow_joint", "wrist_1_joint", "wrist_2_joint", "wrist_3_joint"]
robotBaseFrameName: "ur_base"
robotFlangeFrameName: "ur_flange"
robotTcpFrameName: "ur_flange"
robotReadFrequency: 100
robotWriteFrequency: 100
publishStateFrequency: 100
velocity: 0.1
acceleration: 0.5
angleTolerance: 0.02
maxLinearVelocity: 1.0
maxAngularVelocity: 0.5
